uefi/src/proto/pci/mod.rs

```diff
@@ -8,6 +8,8 @@ use uefi_raw::protocol::pci::root_bridge::PciRootBridgeIoProtocolWidth;
 
 #[cfg(feature = "alloc")]
 pub mod configuration;
+#[cfg(feature = "alloc")]
```

Comment:
there is plenty of functionality in that module not needing alloc

Reply:
now, we should not export private helpers.

I know we are not consistent at all in `uefi` what has to be `alloc`-only and what should only provide additional `alloc`-value add.

ah nvm, You do not have to fix this in this issue. I might create a follow-up issue for that.